pola/logic_ai.py

```diff
@@ -39,5 +39,9 @@ def add_ask_for_pics(product, result):
         ai['ask_for_pics_product'] = 'Uczysz Polę produktu: {}'.format(product.name)
         ai['ask_for_pics_button_start'] = 'Nakręć film'
         ai['max_pic_size'] = 800
-
+        ai["donate"] = {
```

Comment:
Rozumiem, że pole "donate" w tym przypadku jest obiekcie "ai"? Potrzebujemy mieć do niego dostęp wyżej, bo jest to logika niezależna od ai, z którego też chcemy zrezygnować.

Reply:
Fakt. Poprawiam

Reply:
Przeniosłlem.